libs/ml/include/ml/ops/activations/randomized_relu.hpp

```diff
@@ -49,8 +50,34 @@ class RandomizedRelu : public fetch::ml::Ops<T>
     rng_.Seed(random_seed);
     UpdateRandomValue();
   }
+
+  explicit RandomizedRelu(SPType const &sp)
+  {
+    lower_bound_ = sp.lower_bound;
+    upper_bound_ = sp.upper_bound;
+    bounds_mean_ = ((upper_bound_ + lower_bound_) / DataType(2));
```

Comment:
Convention: Please make clear that it's cast and use DataType{2} or static_cast<DataType>(2)